.github/workflows/docs-build.yml

```diff
@@ -1,33 +1,34 @@
 name: Docs Build
 
-on:
-  push:
-    branches:
-      - main
+on: [push]
```

Comment:
should be restrict to **not** run on `main`? otherwise the build will always be broken

Reply:
I think we can skip main. Why would it be broken though? I think it might be wasteful but it should noop on main since it ran on the branhc.

Reply:
Won't the `push` on main break?

Reply:
Not sure! I find actions pretty confusing ;)

Reply:
Updated in 4c7121e0db6755e4d1b00c7aa5ab5b35fc7763f6